hub3/ead/ead.go

```diff
@@ -233,32 +233,48 @@ func (dsc *Cdsc) NewNodeList(cfg *NodeConfig) (*NodeList, uint64, error) {
 		nl.Label = append(nl.Label, label.Head)
 	}
 
-	for _, cc := range dsc.Numbered {
-		node, err := NewNode(cc, []string{}, cfg)
+	nodes := make([]*Node, 0)
+	errors := make([]string, 0)
+
+	for _, p := range dsc.Cp {
+		cc := p.NewClevel()
+		node, err := NewNode(CLevel(cc), []string{}, cfg)
 		if err != nil {
-			return nil, 0, err
```

Comment:
For big EADs it is important to fail early and give the error. The handling of cLevels is already fairly robust and errors that are thrown in creating of a ead.Node are fatal for proper processing.

Reply:
Returned early on all three loops